README.md

```diff
@@ -5,12 +5,12 @@
 --><a href="https://colab.research.google.com/github/bellingcat/EDGAR/blob/main/notebook/Bellingcat_EDGAR_Tool.ipynb"><img alt="Colab icon: Try it on Colab" src="https://img.shields.io/badge/Try%20it%20on%20Colab-%20?style=for-the-badge&logo=googlecolab&logoColor=fff&logoSize=auto&color=e8710a">
 </a>
 
-Python tool to search and retrieve corporate and financial data from the United States Securities and Exchange Commission (SEC). 
+Python tool to search and retrieve corporate and financial data from the United States Securities and Exchange Commission (SEC).
```

Comment:
:( `black` ran on the README after I added the new development workflow. Those are the only changes here.

Reply:
Trailing whitespace 🫡